checkstyle-suppressions.xml

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE suppressions PUBLIC
+        "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+        "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+    <suppress checks="OperatorWrap" files=".*" />
+    <suppress checks="JavadocParagraph" files=".*" />
+    <suppress checks="SummaryJavadoc" files=".*" />
+    <!-- suppress check for protobuf-generated files. -->
+    <!-- for windows separator-->
+    <suppress checks=".*" files="src[\\\\]main[\\\\]java[\\\\]com[\\\\]distkv[\\\\]rpc[\\\\]protobuf[\\\\]generated" />
```

Comment:
???
 You should take a look at what this code does. What's the point of copying it here?

Reply:
ok